### PR TITLE
Pytube (compatible to python 3)

### PR DESCRIPTION
**Pull request by lukecold**
Added compatibility to python 3 (plus, the test does not cover it)

### CODE REVIEW

**Comment by nficano** [comment]
Great! I'll check this out this evening and merge once reviewed! thank you sir!!!

**Comment by lukecold** [comment]
I'd thank you for this awesome tool! It's always nice to have the chance to contribute to open source software. :)

File: pytube/api.py [hunk 1]
```diff
@@ -322,16 +340,18 @@ def _get_json_offset(self, html):
         :param str html:
             The raw html of the YouTube page.
         """
-        brackets = []
+        unmatched_brackets_num = 0
         index = 1
         # Determine the offset by pushing/popping brackets until all
         # js expressions are closed.
         for idx, ch in enumerate(html):
+            if(type(ch) == type(50)):
```
Comment (nficano) [inline pytube/api.py:348]: What is the case when ch is type(50)?

Comment (lukecold) [inline pytube/api.py:348]: 50 is just a random number, it means type of integer

**Pull request merged**